drafts/LightningCreations/LCSD2/LCSD2.tex

```diff
@@ -0,0 +1,138 @@
+\section{Lightning Creations LCSD 2 - Proposed Procedure for LCS Draft
```

Comment:
Since this file will be generated from the MarkDown, I will be exclusively reviewing that version of the document.